docs/python/python-on-azure.md

```diff
@@ -1,5 +1,5 @@
 ---
-Order: 13
+Order: 12
```

Comment:
You don't need to revert these changes but, in the future, you can save yourself time and leave the Order values the same. It's OK to have gaps in the Order values.

Reply:
oh LOL thanks Greg!